cfme/tests/optimize/test_bottlenecks.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+import fauxfactory
+import pytest
+from datetime import timedelta
+
+from fixtures.pytest_store import store
+from cfme.optimize.bottlenecks import Bottlenecks
+from utils import conf
+from utils.appliance.implementations.ui import navigate_to
+from utils.appliance import get_or_create_current_appliance
+from utils.ssh import SSHClient
+from utils.version import current_version
+
+db_tbl = store.current_appliance.db['bottleneck_events']
+db_events = store.current_appliance.db.session.query(db_tbl.timestamp,
+    db_tbl.resource_type, db_tbl.resource_name, db_tbl.event_type, db_tbl.severity, db_tbl.message)
+
+
+@pytest.fixture(scope="session")
+def extend_db_partition():
+    app = get_or_create_current_appliance()
```

Comment:
Here is the sprout provisioning fn: https://github.com/ManageIQ/integration_tests/blob/master/cfme/test_framework/sprout/client.py#L96

Please use it here and I'll be switching the `test_db_migrate` test to do the same.